design/key_derivation.md

```diff
@@ -0,0 +1,132 @@
+# Key Derivation design
+
+## Scope
+
+The Obscuro blockchain has a network key that allows read/write to the transaction in the block chain.
+This network key is shared between enclaves upon successful attestation.
+
+Transactions are encrypted in the rollups that make the Obscuro blockchain.
+Each rollup uses a key to encrypt the Transactions.
+Each key is derived from the initial network Key.
```

Comment:
it's derived from the master seed which is not a key

Reply:
gotcha